src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php

```diff
@@ -100,4 +111,16 @@ private function getLanguageName(string $code): ?string
             return null;
         }
     }
+
+    private function updateLocaleParameter(string $code): void
+    {
+        if ($this->filesystem->exists($this->localeParameterFilePath) === false) {
+            throw new \RuntimeException(sprintf('File %s does not exist', $this->localeParameterFilePath));
+        }
+
+        $content = Yaml::parseFile($this->localeParameterFilePath);
+        $content['parameters']['locale'] = $code;
+
+        $this->filesystem->dumpFile($this->localeParameterFilePath, Yaml::dump($content));
```

Comment:
Changing the locale in the file can be treated as a new feature - at least it's definitely a new behaviour of this class. I would not say it should prevent us from that change (it's still a dev-only one), but let's maybe mention it in the UPGRADE file to be 100% transparent 🖖